### ISSUE TITLE
some question about haze Removal

### ISSUE THREAD

**Issue by liang233** [liu-vis/DualResidualNetworks#4]
Thanks for good work.but I have a question,why you do not use norm layer in  DuRN-US network in haze removal and use bacthnorm in rain steak removal?

**Comment by liu-vis** [comment]
Hi, thank you for your interest in our work. 
The haze effects make the average intensity of an image higher, but I think they don't really degrade the image that hard. In other words, it is easier to recover an image to a good extent from the haze effects, than from other factors (e.g., motion-blur, rain-streak, raindrop, etc.).
For this reason, I think a CNN trained for haze removal doesn't really suffer from the internal covariate shift problem [(batch norm paper link)](https://arxiv.org/pdf/1502.03167.pdf) that much, and thus no need to normalize the layers' outputs.

**Comment by liang233** [comment]
hello，How much is the batchsize chosen for RESIDE? I only have a 1080ti，I find it is not converge.

**Comment by liu-vis** [comment]
Hi, sorry for the late reply. 
My batchsize is 48 for 4-GPUs(1080ti).
Setting batchsize = 10 for single 1080ti should be totally fine for training.
